ddvk/remarkable2-framebuffer
# [client] support xochitl 2.6

this diff refactors the signature check and allows for multiple signatures per function.
hopefully the signatures won't change often. this diff updates the signatures for 2.6 as well

test plan:

* start xochitl using `LD_PRELOAD=...librm2fb_client.so.1.0.1`
* verify there are no "function not found" messages in log
* verify xochitl displays properly and can be interacted with
